logfire/_internal/auto_trace/__init__.py

```diff
@@ -24,13 +23,7 @@ def install_auto_tracing(
 
     See `Logfire.install_auto_tracing` for more information.
     """
-    if modules is None:
-        frame = inspect.stack()[2]
-        module = inspect.getmodule(frame[0])
-        if module is None:  # pragma: no cover
-            raise KeyError('module not found')
-        modules = modules_func_from_sequence([module.__name__.split('.')[0]])
-    elif isinstance(modules, Sequence):  # pragma: no branch
+    if isinstance(modules, Sequence):  # pragma: no branch
         modules = modules_func_from_sequence(modules)
 
     if not callable(modules):  # pragma: no cover
```

Comment:
Maybe `if` / `elif` here? This seems like a prime opportunity to add tests for callable and also wrong-type inputs.

```suggestion
    if isinstance(modules, Sequence):
        modules = modules_func_from_sequence(modules)

    elif not callable(modules):
```

Reply:
Added a test for the wrong type. Testing a callable feels a bit redundant. Not using elif to make it clear that the sequence gets converted to a callable, so `modules` is always eventually a callable.